desktop/components/_buttons.scss

```diff
@@ -153,7 +153,7 @@ category: Buttons
 
     &.btn--facebook {
         background-color: $colorFacebook;
-        background-image: url("data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIyMiIgaGVpZ2h0PSI0MCIgdmlld0JveD0iMCAwIDIyIDQwIj48cGF0aCBmaWxsPSJ3aGl0ZSIgZD0iTTIwLjgxMi4wMDhMMTUuNjI1IDBDOS43OTcgMCA2LjAzIDMuODY0IDYuMDMgOS44NDR2NC41NEguODE2Yy0uNDUgMC0uODE1LjM2NS0uODE1LjgxNXY2LjU3NmMwIC40NS4zNjUuODE1LjgxNS44MTVINi4wM3YxNi41OTVjMCAuNDUuMzY1LjgxNS44MTYuODE1aDYuODA1Yy40NSAwIC44MTYtLjM2NS44MTYtLjgxNVYyMi41OWg2LjA5OGMuNDUgMCAuODE2LS4zNjQuODE2LS44MTRsLjAwMy02LjU3N2MwLS4yMTctLjA4Ny0uNDI0LS4yNC0uNTc3LS4xNTItLjE1NC0uMzYtLjI0LS41NzYtLjI0aC02LjF2LTMuODQ4YzAtMS44NS40NC0yLjc4OCAyLjg1LTIuNzg4aDMuNDk0Yy40NTIgMCAuODE2LS4zNjYuODE2LS44MTZWLjgyNWMwLS40NS0uMzY0LS44MTUtLjgxNC0uODE2eiIvPjwvc3ZnPg==");
+        background-image: url("/images/local_mobile/svg_icons/icon_fb_white_logo.svg");
```

Comment:
is it okay if I put it in /images/icon_fb_white_logo.svg ?
 I'm going to use it for both desktop and webmobile